server/i18n/en.json

```diff
@@ -7402,14 +7406,6 @@
     "id": "ent.compliance.licence_disable.app_error",
     "translation": "Compliance functionality disabled by current license. Please contact your system administrator about upgrading your enterprise license."
   },
-  {
```

Comment:
It looks like some of these translations got removed accidentally due to a bad merge. Could you re-add those?

Reply:
Yes, you are right, those were removed by accident. I re-added the translations.